app/views/idv/otp_verification/show.html.erb

```diff
@@ -45,7 +45,7 @@
       outline: true,
       icon: :loop,
       class: 'margin-bottom-4',
-    ).with_content(t('links.two_factor_authentication.get_another_code')) %>
+    ).with_content(t('links.two_factor_authentication.send_another_code')) %>
```

Comment:
IMO I don't think the change is big enough to merit renaming the key? But also fine to change it too

Reply:
Yeah, I'm not sure where that line is.